students/fahad/Lab 2/Part_1/Part_1.ino

```diff
@@ -0,0 +1,41 @@
+/**
+   Function: customPWM
+
+   Pin: Pin number to execute PWM on
+   dutyCycle: Percentage of brightness. 0 is off and 100 is maximum brightness
+   duration: how long the LED should be on for in microseconds
+*/
+const int bLED = 8;
+void customPWM (int pin, float dutyCycle, double duration) {
+
+  static const int period = 10000;
+  float brightness;
+  float T_on, T_off;
+
+  brightness = dutyCycle * 10;
+  T_on = (brightness * period) / 1000;
```

Comment:
Simplify:
```suggestion
  T_on = (dutyCycle * period) / 100;
```
Or if you make `dutyCycle` between 0 and 1, you don't even need to divide by 100.